.github/workflows/main.yaml

```diff
@@ -17,13 +17,13 @@ concurrency:
 jobs:
 
   build:
-    uses: ./.github/workflows/build-rock.yaml
+    uses: canonical/identity-credentials-workflows/.github/workflows/build-rock@main
```

Comment:
A problem I see with those workflows is that there are not versioned, I think we should follow a similar route than [sdcore-github-workflows](https://github.com/canonical/sdcore-github-workflows) using semantic versioning would allow to include breaking changes to workflows without breaking all of our CI's at once.

Reply:
yeah that's fair. I was trying to keep these pretty simple to start, but that seems like a worthwhile addition

Reply:
fixed in https://github.com/canonical/lego-rock/pull/44/commits/9bdbf2151836b5b27b7d95675df967d241546e87